src/cider/tests/nextgen/functional/CiderMathOpNextgenTest.cpp

```diff
@@ -56,157 +58,152 @@ class CiderMathOpNullableArrowTest : public CiderTestBase {
 
 TEST_F(CiderMathOpNullableArrowTest, ConstantValueMathOpTest) {
   GTEST_SKIP();
```

Comment:
Seems there is some unexpected `GTEST_SKIP` in this file, can you help remove them except the `Decimal `one?